YearInReview/Model/Reports/ReportGenerator.cs

```diff
@@ -55,8 +58,26 @@ public async Task<Report1970> Generate(int year)
 			var games = _playniteApi.Database.Games;
 			var activities = await _gameActivityExtension.GetActivityForGames(games);
 			var filteredActivities = _specificYearActivityFilter.GetActivityForYear(year, activities);
+			
+			ShowReportGeneratedNotification(year);
 
 			return _composer1970.Compose(year, filteredActivities);
 		}
+
+		private void ShowReportGeneratedNotification(int year)
+		{
+			var description = string.Format(
+				ResourceProvider.GetString("LOC_YearInReview_ReportGeneratedNotification"), 
+				year
+			);
+			
+			_playniteApi.Notifications.Add(
+				new NotificationMessage(
+					NotificationId + year,
+					description,
+					NotificationType.Info
+				)
```

Comment:
I like the idea about adding an action. The official way to do it would be to show our separate window report. 

If you're feeling adventurous, there are some hacks the StartPage uses to push it's sidebar item to be at the top of the list: https://github.com/felixkmh/StartPage-for-Playnite/blob/7db24ccec9bc7121fd4e7f8d58f07747b08a17a4/source/StartPage/LandingPageExtension.cs#L302

Looks like WPF code, so we probably would be able to click our sidebar item to activate it.

This is not supported by Playnite.SDK, and we should probably avoid doing it, but if we add some optional setting that enables this logic, it should be fine 😅 .

Either way, I suggest to merge this PR, and work on a new PR, so you don't have long hanging changes.